spec/support/seeds/e2e/admin_login_setup.rb

```diff
@@ -0,0 +1,39 @@
+return if Rails.env.production?
+
+require Rails.root.join("app/models/site_config")
+
+# NOTE: when adding new data, please use the Seeder class to ensure the seed tasks
```

Comment:
Now that I think about it more, is this really a requirement for CI? We made `db/seeds.rb` idempotent in local envs so people can selectively nuke certain models and re-run the script without errors, this doesn't seem necessary on CI, does it?

Reply:
It's not so much a requirement for CI as it is for E2E tests. I get the idempotent part of the Seeder class, but for the end to end tests, we need specific data set up. The person writing the end to end tests could nuke models in the current seeded data, but in that case, it would still require a script to do that which would need to be run at the beginning of the E2E test which is why I currently have the current rake E2E task.

Maybe this is really only an issue with the first test I have that creates the initial Forem instance admin, i.e. I need a pretty empty database.

After that It might be fine to just use the existing seeded data like you mention, `bundle exec rake db:seed`, which I can call instead in my Cypress task `seedData`.

Reply:
I was primarily thinking out loud, as I'm not fully aware of all the requirements you have around E2E tests. Please do whatever works for you, but feel free to hit me up on Slack or via a Zoom call if you want to hash out some details.

Reply:
It got me thinking out loud. 😄